ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/picklist/activityPicklistAnswer.component.ts

```diff
@@ -6,39 +6,62 @@ import { PicklistRenderMode } from './../../../models/activity/picklistRenderMod
-        </ddp-activity-radiobuttons-picklist-question>
-        <ddp-activity-checkboxes-picklist-question
-            *ngIf="block.selectMode === SELECT_MODE.MULTIPLE && block.renderMode === RENDER_MODE.LIST || block.renderMode === RENDER_MODE.CHECKBOX_LIST"
-            [block]="block"
-            [readonly]="readonly"
-            (valueChanged)="valueChanged.emit($event)">
-        </ddp-activity-checkboxes-picklist-question>
-        <ddp-activity-dropdown-picklist-question
-            *ngIf="block.renderMode === RENDER_MODE.DROPDOWN"
-            [block]="block"
-            [readonly]="readonly"
-            (valueChanged)="valueChanged.emit($event)">
-        </ddp-activity-dropdown-picklist-question>
-        <ddp-activity-autocomplete-picklist-question
-            *ngIf="block.renderMode === RENDER_MODE.AUTOCOMPLETE"
-            [block]="block"
-            [readonly]="readonly"
-            (valueChanged)="valueChanged.emit($event)">
-        </ddp-activity-autocomplete-picklist-question>
-    </div>`
+    template: ` <ddp-question-prompt [block]="block"></ddp-question-prompt>
+        <div>
+            <ddp-activity-radiobuttons-picklist-question
+                *ngIf="
+                    block.selectMode === SELECT_MODE.SINGLE &&
+                    block.renderMode === RENDER_MODE.LIST
+                "
+                [block]="block"
+                [readonly]="readonly"
+                (valueChanged)="valueChanged.emit($event)"
+            >
+            </ddp-activity-radiobuttons-picklist-question>
+            <ddp-activity-checkboxes-picklist-question
+                *ngIf="
+                    (block.selectMode === SELECT_MODE.MULTIPLE &&
+                        block.renderMode === RENDER_MODE.LIST) ||
+                    block.renderMode === RENDER_MODE.CHECKBOX_LIST
+                "
+                [block]="block"
+                [readonly]="readonly"
+                (valueChanged)="valueChanged.emit($event)"
+            >
+            </ddp-activity-checkboxes-picklist-question>
+            <ddp-activity-dropdown-picklist-question
+                *ngIf="block.renderMode === RENDER_MODE.DROPDOWN"
+                [block]="block"
+                [readonly]="readonly"
+                (valueChanged)="valueChanged.emit($event)"
+            >
+            </ddp-activity-dropdown-picklist-question>
+            <!-- <ddp-activity-autocomplete-picklist-question
+                *ngIf="block.renderMode === RENDER_MODE.AUTOCOMPLETE"
+                [block]="block"
+                [readonly]="readonly"
+                (valueChanged)="valueChanged.emit($event)"
+            >
+            </ddp-activity-autocomplete-picklist-question> -->
+            <ddp-activity-picklist-remote-auto-complete-options
+                *ngIf="block.renderMode === RENDER_MODE.AUTOCOMPLETE"
+                [block]="block"
+                [readonly]="readonly"
+                [studyGuid]="studyGuid"
```

Comment:
Here is being injected to ddp-activity-picklist-remote-auto-complete-options component